calendar_app/enums.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class Status(Enum):
+    INIT = 0
+    PENDING = 1
+    FINISHED = 2
+    FAILED = 3
```

Comment:
failed/wrong/bad statuses can be -1, -2 , -3 then positive values will be always correct